tests/ui/lexer/lex-bad-str-literal-as-char-3.rs

```diff
@@ -1,4 +1,8 @@
-//@ run-rustfix
+//@ revisions: rust2015 rust2018 rust2021
+//@[rust2018] edition:2018
+//@[rust2021] edition:2021
 fn main() {
-    println!('hello world'); //~ ERROR unterminated character literal
+    println!('hello world');
+    //[rust2015,rust2018,rust2021]~^ ERROR unterminated character literal
```

Comment:
Could be simplified to

```suggestion
    //~^ ERROR unterminated character literal
```

but not blocking.